examples/Migration/CampaignManagement/CreateCompleteCampaignGoogleAdsApiOnly.php

```diff
@@ -0,0 +1,499 @@
+        string $customerId,
+        string $adGroupResourceName
+    ) {
+        $googleAdsServiceClient = $googleAdsClient->getGoogleAdsServiceClient();
+        $query = "SELECT ad_group.id, ad_group.name, " .
+                 "ad_group.resource_name from ad_group where " .
+                 "ad_group.resource_name='$adGroupResourceName'";
+
+        $response =
+            $googleAdsServiceClient->search($customerId, $query, ['pageSize' => self::PAGE_SIZE]);
+
+        return $response->getIterator()->current()->getAdGroup();
+    }
+
+    /**
+     * Creates text ads.
+     * @param GoogleAdsClient $googleAdsClient the Google Ads API client
+     * @param string $customerId the client customer ID without hyphens
+     * @param AdGroup $adGroup the ad group
+     * @return AdGroupAd[] an array of text ads
+     */
+    private static function createTextAds(
+        GoogleAdsClient $googleAdsClient,
+        string $customerId,
+        AdGroup $adGroup
+    ) {
+        $adGroupResourceName = $adGroup->getResourceName();
+
+        $operations = [];
+        for ($i = 0; $i < self::NUMBER_OF_ADS; $i++) {
+            // Creates the expanded text ad info.
+            $expandedTextAdInfo = new ExpandedTextAdInfo([
+                'headline_part1' => new StringValue(['value' => 'Cruise to Mars #' . uniqid()]),
+                'headline_part2' => new StringValue(['value' => 'Best Space Cruise Line']),
+                'description' => new StringValue(['value' => 'Buy your tickets now!'])
+            ]);
+
+            // Sets the expanded text ad info on an Ad.
+            $ad = new Ad([
+                'expanded_text_ad' => $expandedTextAdInfo,
+                'final_urls' => [new StringValue(['value' => 'http://www.example.com'])]
+            ]);
+
+            // Creates an ad group ad to hold the above ad.
+            $adGroupAd = new AdGroupAd([
+                'ad_group' => $adGroupResourceName,
+                'status' => AdGroupAdStatus::PAUSED,
+                'ad' => $ad
+            ]);
+
+            // Creates an ad group ad operation and add it to the operations array.
+            $adGroupAdOperation = new AdGroupAdOperation();
+            $adGroupAdOperation->setCreate($adGroupAd);
+            $operations[] = $adGroupAdOperation;
+        }
+
+        // Issues a mutate request to add the ad group ads.
+        $adGroupAdServiceClient = $googleAdsClient->getAdGroupAdServiceClient();
+        $adGroupAdResponse = $adGroupAdServiceClient->mutateAdGroupAds($customerId, $operations);
+
+        $newAdResourceNames = [];
+        foreach ($adGroupAdResponse->getResults() as $result) {
+            $newAdResourceNames[] = $result->getResourceName();
+        }
+
+        $newAds = self::getAds($googleAdsClient, $customerId, $newAdResourceNames);
+        foreach ($newAds as $newAd) {
+            printf(
+                "Created expanded text ad with id '%d', status %d and headline '%s - %s'.%s",
```

Comment:
id doesn't need to be in single quote and it should be spelled as ID

Reply:
Done.